portal-web/docroot/html/taglib/ui/input_asset_links/page.jsp

```diff
@@ -32,7 +32,7 @@
 			cssClass="asset-selector"
 			data='<%= (Map<String, Object>)selectorEntry.get("data") %>'
 			id='<%= (String)selectorEntry.get("id") %>'
-			message='<%= (String)selectorEntry.get("message") %>'
+			message='<%= HtmlUtil.escape((String)selectorEntry.get("message")) %>'
```

Comment:
This causes a double escape in /portal-web/docroot/html/taglib/ui/icon/page.jsp:94
1. Do the escaping in /portal-web/docroot/html/taglib/ui/icon/page.jsp
2. Do a complete review of every use case of <liferay-ui:icon> and if there's an escape on the `message` attribute, you'll need to remove it.


@daviddotzhang